Some changes to UPGRADE docs (#1749)
digitallyinduced/ihp: UPGRADE.md

    After that, your `start` script is not needed anymore, and you can delete it.

4. **Optional: Delete `start` script**

>     The `start` script is not needed anymore, and can be deleted. The new start command is now `devenv up`. You can keep `start` script for b.c. purposes if you want to.

Reviewer: ```suggestion
    The `start` script is not needed anymore, and can be deleted. The new start command is now `devenv up`. You can keep the `start` script if you want to, but it's not required.
```
